GoogleForCreators/web-stories-wp
# Separate global and local color presets

## Summary
- Adds separate local color presets section to the Saved colors panel


## Relevant Technical Choices



## To-do

- [ ] Review styling



## User-facing changes
Before:
<img width="304" alt="Screenshot 2021-01-22 at 16 09 08" src="https://user-images.githubusercontent.com/3294597/105500924-dfea8980-5cc3-11eb-8fe5-823eb4ae5df6.png">

After:
<img width="304" alt="Screenshot 2021-01-22 at 16 09 30" src="https://user-images.githubusercontent.com/3294597/105500948-e37e1080-5cc3-11eb-84fa-3a7e29513381.png">



## Testing Instructions
1. Add a shape
2. Open the Saved Colors panel and delete all existing colors (if any).
3. Verify the view matches the view in Figma and has two separate buttons for adding colors for Current / All Stories.
4. Click to add a color for the current story.
5. Verify that two sections appear, one for the current story, one for all stories.
6. Verify that the color is added correctly under the right section based on which button is clicked.
7. Verify that removing colors works as expected
8. Verify that only the colors saved for "All stories" are visible for other stories, the color for the current story should be empty by default when adding a new story.


---



Fixes #3635
